### ISSUE TITLE
Is it okay to commit jwt_secret? 

### ISSUE THREAD

**Issue by jadjoubran** [tymondesigns/jwt-auth#311]
Shouldn't the `jwt_secret` be moved to `.env`?
If so, the `artisan jwt:generate` needs to be fixed as well

**Comment by tdhsmith** [comment]
The `jwt:generate` command can be run with flag `--show` to output the value directly (which you can then paste into your `.env`). I could've sworn this became the default at some point, but it doesn't appear to be now.

I agree that the secret should not get committed to version control. The current config value _does_ read from the env file though, so it would just be a matter of changing either the generate function or the documentation...

**Comment by tymondesigns** [comment]
That's correct, you can simply drop your secret into your .env file across your environments. The generate command is just a convenience really, but I get that the docs should be updated to make that clearer.

What are your thoughts on changing the command so that it updates the .env if one exists, instead of updating the fallback in the config file directly?

**Comment by tymondesigns** [comment]
Also I was considering changing the command name to `jwt:secret` or something, maybe with an extra command to maintain backwards compatibility

**Comment by jadjoubran** [comment]
@tymondesigns I think it makes sense to update the command since people assume that what they just installed is following best practices
So if you put it in the fallback, it sounds as if it's better than `.env`
I'd be happy to send a pull request if you wish. I just need to know the details (command name: new/old? - additional options?)

Thanks for your quick reply

**Comment by tymondesigns** [comment]
Ok, I think I've sorted it.

The behaviour now is to check if a `JWT_SECRET`` has been set in the `.env` file. 
If it has then it will replace the key with a new one, otherwise it will add the key to the end of the file.

Oh and I've renamed the command to `jwt:secret` as I think it makes more sense symantically.

What do you guys think? [link](https://github.com/tymondesigns/jwt-auth/blob/develop/src/Commands/JWTGenerateSecretCommand.php)

**Comment by jadjoubran** [comment]
Sounds great, thanks! :D
